Makefile

```diff
@@ -426,26 +426,26 @@ docker-buildx-dev: docker-buildx-builder
 		$(if $(filter true,$(PUSH_TO_GHCR)),-t $(GHCR_REPOSITORY):$(DEV_TAG)) \
 		$(if $(filter true,$(PUSH_TO_GHCR)),-t $(GHCR_REPOSITORY):dev) .
 
-docker-buildx-crds-dev: build-crds docker-buildx-builder
+docker-buildx-release: docker-buildx-builder
 	docker buildx build \
 		$(_ATTESTATIONS) \
 		--build-arg LDFLAGS=${LDFLAGS} \
 		--platform="$(PLATFORM)" \
 		--output=$(OUTPUT_TYPE) \
-		-t $(CRD_REPOSITORY):$(DEV_TAG) \
-		-t $(CRD_REPOSITORY):dev \
-		$(if $(filter true,$(PUSH_TO_GHCR)),-t $(GHCR_CRD_REPOSITORY):$(DEV_TAG)) \
-		$(if $(filter true,$(PUSH_TO_GHCR)),-t $(GHCR_CRD_REPOSITORY):dev) \
-		-f crd.Dockerfile .staging/crds/
+		-t $(REPOSITORY):$(VERSION) \
+		$(if $(filter true,$(PUSH_TO_GHCR)),-t $(GHCR_REPOSITORY):$(VERSION)) .
 
-docker-buildx-release: docker-buildx-builder
+docker-buildx-crds-dev: build-crds docker-buildx-builder
```

Comment:
Nit: This rule is getting swapped with `docker-buildx-release` and vice versa without any change.